shardingsphere-features/shardingsphere-primary-replica/shardingsphere-primary-replica-api/src/main/java/org/apache/shardingsphere/primaryreplica/api/config/PrimaryReplicaRuleConfiguration.java

```diff
@@ -15,25 +15,25 @@
  * limitations under the License.
  */
 
-package org.apache.shardingsphere.masterslave.api.config;
+package org.apache.shardingsphere.primaryreplica.api.config;
 
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 import org.apache.shardingsphere.infra.config.RuleConfiguration;
 import org.apache.shardingsphere.infra.config.algorithm.ShardingSphereAlgorithmConfiguration;
-import org.apache.shardingsphere.masterslave.api.config.rule.MasterSlaveDataSourceRuleConfiguration;
+import org.apache.shardingsphere.primaryreplica.api.config.rule.PrimaryReplicaDataSourceRuleConfiguration;
 
 import java.util.Collection;
 import java.util.Map;
 
 /**
- * Master-slave rule configuration.
+ * Primary-replica rule configuration.
  */
 @RequiredArgsConstructor
 @Getter
-public final class MasterSlaveRuleConfiguration implements RuleConfiguration {
+public final class PrimaryReplicaRuleConfiguration implements RuleConfiguration {
```

Comment:
The rule name should be ReadWriteSplitRuleConfiguration

Reply:
Will change